lib/ansible/modules/cloud/vmware/vmware_spbm.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name, too-many-instance-attributes, broad-except
+# Prasanna Nanda <pnanda@cloudsimple.com>
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+"""Manages Storage Profiles"""
+
+from ansible.module_utils.vmware import vmware_argument_spec, connect_to_api, find_datastore_by_name
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils._text import to_native
+from ansible.module_utils.spbm_client import SPBMClient
+
+try:
```

Comment:
Can be replaced with 
https://github.com/ansible/ansible/blob/a3390b63fe317eb2fea5d78fb99c8a5e8fc8eaf3/lib/ansible/modules/cloud/vmware/vmware_host_ntp.py#L93